<launch>
	
	<!-- Publish the static transform between the robot and laser -->
	<node pkg="tf" type="static_transform_publisher" name="base_link_laser_publisher" args="0.0 0.0 1.0 0.0 0.0 0.0 base_link laser 100"/>
	
	<rosparam command="load" file="$(find rse_project)/config/scan_params.yaml"/>
	<rosparam command="load" file="$(find rse_project)/config/tf_params.yaml"/>
	<!-- Run the node to publish laser scan and dynamic transform -->
	<node pkg="rse_project" type="scan_with_tf_publisher" name="scan_with_tf_publisher" output="screen"/>

	<!-- Run rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find rse_project)/config/rviz_config.rviz"/>
	
</launch>
